lib/mapper.js

```diff
@@ -142,8 +142,9 @@ function formatRemarketingTag(msg) {
     bundleid: app.namespace,
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
     remarketing_only: 1,
+    usage_tracking_enabled: 1,
```

Comment:
https://developers.google.com/app-conversion-tracking/ios/remarketing-server

"usage_tracking_enabled=1 (required)

This tells AdWords that you are sending app usage data, so that certain types of remarketing lists can be made available."

Seems to just be another required setting that we haven't been setting.